makefile

```
@@ -47,7 +47,11 @@ lint-shell:
     || exit 1; done \
   fi
 
-lint: lint-python lint-shell lint-swagger lint-docker lint-yaml
+lint-helm:
+	helm lint ./helm
```

Consider making the checks more strict:

```suggestion
	helm lint --strict ./helm
```